sdk/program/src/instruction.rs

```diff
@@ -253,9 +253,9 @@ pub enum InstructionError {
     #[error("Account data allocation exceeded the maximum accounts data size limit")]
     MaxAccountsDataSizeExceeded,
 
-    /// Active vote account close
-    #[error("Cannot close vote account unless it stopped voting at least one full epoch ago")]
-    ActiveVoteAccountClose,
+    /// Max accounts exceeded
+    #[error("Max accounts exceeded")]
+    MaxAccountsExceeded,
```

Comment:
What about [sdk/program/src/program_error.rs](https://github.com/solana-labs/solana/blob/master/sdk/program/src/program_error.rs)?

Reply:
Ah thanks, will fix

Reply:
Actually, there is no need to add a program error variant because this error is only for outside the vm. We can add support for programs returning this error later if it's needed.